content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +81,193 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
 
         $ sudo systemctl start rippled.service
 
-## Installation on Ubuntu with alien
+For next steps, see [Postinstall](#postinstall) and [Additional Configuration](#additional-configuration).
 
-This section assumes that you are using Ubuntu 15.04 or later.
 
-1. Install yum-utils and alien:
+## Installation on macOS
 
-        $ sudo apt-get update
-        $ sudo apt-get install yum-utils alien
+At this time, we don't recommend using the macOS platform for `rippled` production use. For production, consider using the [Ubuntu platform](#installation-on-ubuntu-with-alien), which has received the highest level of quality assurance and testing.
 
-2. Install the Ripple RPM repository:
+That said, macOS is suitable for many development and testing tasks.
 
-        $ sudo rpm -Uvh https://mirrors.ripple.com/ripple-repo-el7.rpm
+This section assumes that you are using macOS X 10.0 or higher. ***TODO: okay?***
 
-3. Download the `rippled` software package:
+{% set n = cycler(* range(1,99)) %}
 
-        $ yumdownloader --enablerepo=ripple-stable --releasever=el7 rippled
+{{n.next()}}. Install [Xcode](https://developer.apple.com/download/).
 
-4. Verify the signature on the `rippled` software package:
+{{n.next()}}. Install Xcode command line tools. ***TODO: Comment from PM: Installing homebrew removes the need to install xcode command line tools separately. JHA: homebrew lists xcode command line tools as a requirement: https://docs.brew.sh/Installation#requirements. Do we keep or remove this step?***
```

Comment:
Ah, maybe its dependent on some sort of installation settings where you can install both simultaneously. I think its safer bet to leave it as for now, I can look into it.